pegasus/sites.v3/code.org/views/global/fa/farsiheader.haml

```diff
@@ -30,7 +35,7 @@
       %img.logo{src: '/images/logo.svg', alt: I18n.t(:code_org_logo_alt)}
```

Comment:
The only real need we have for the separate farsiheader file is currently the logo. We can try to add this to the region configuration instead, and start to get rid of these separate per-region files we probably do not want to maintain forever.